templates/search/blocks/search_pagination.html

```diff
@@ -7,7 +7,7 @@
                     <ul class="pagination__list">
                         {% if page.has_previous %}
                             <li class="pagination__list-item">
-                                <a class="pagination__page-chevron-previous" href="?{% query_string_include 'page' page.previous_page_number %}" data-link-type=”Pagination”><span
+                                <a class="pagination__page-chevron-previous" href="?{% query_string_include 'page' page.previous_page_number %}" data-link-type="Pagination"><span
                                         class="sr-only" data-link="Previous page">Previous page</span></a>
```

Comment:
Hi John, any chance data-link="Previous page" can be moved inline with data-link-type?